hw9/ArrayValueCalculator.java

```diff
@@ -0,0 +1,24 @@
+package org.hillel.hw9;
+
+import org.hillel.hw9.exceptions.ArrayDataException;
+import org.hillel.hw9.exceptions.ArraySizeException;
+
+public class ArrayValueCalculator {
+    public static int doCalc(Object[][] arr) throws ArraySizeException, ArrayDataException {

```

Comment:
`throws ArraySizeException, ArrayDataException` should be used if you wanna delegate error-handling to the end user, but it seems that you're validating the input on your own, so throws is redundant here, remove it.